### PR TITLE
migrate code from googleapis/nodejs-video-stitcher

### PR DESCRIPTION
**Pull request by irataxy**
- feat: add templated files from docker image
- feat: add initial samples (#2)
- docs(samples): add Video Stitcher slate and CDN key code samples and tests (#6)
- docs(samples): add Video Stitcher VOD and live session code samples a… (#9)
- docs(samples): add try/catch to delete operations in testing (#19)
- chore(main): release 0.1.1 (#16)
- chore(main): release 0.2.0 (#21)
- chore(main): release 0.2.1 (#25)
- chore(main): release 0.2.2 (#29)
- docs(samples): Fix Video Stitcher region tag prefix (#36)
- chore(main): release 0.3.0 (#30)

### CODE REVIEW

File: media/video-stitcher/quickstart.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+'use strict';
+
+function main(parent) {
+  // [START videostitcher_v1_generated_VideoStitcherService_ListCdnKeys_async]
```
Comment (ace-n) [inline media/video-stitcher/quickstart.js:22]: This sample isn't used in the docs. Do we need it here?

File: media/video-stitcher/test/quickstart.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+'use strict';
+
+const path = require('path');
```
Comment (ace-n) [inline media/video-stitcher/test/quickstart.js:20]: Do we need this file, if `quickstart.js` is itself not in the docs?

(If we do, this file should be named `quickstart.test.js`.)

**Review by ace-n — CHANGES_REQUESTED** [review]

File: media/video-stitcher/test/quickstart.js (hunk 2 above)
Comment (irataxy) [reply media/video-stitcher/test/quickstart.js]: Removed

File: media/video-stitcher/quickstart.js (hunk 1 above)
Comment (irataxy) [reply media/video-stitcher/quickstart.js]: Removed

**Review by ace-n — APPROVED** [review]
LGTM once tests pass

**Pull request merged**